cli/src/main/kotlin/io/spine/protodata/cli/app/FileSetOptionsProvider.kt

```diff
@@ -24,21 +24,32 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.protodata.test.annotation;
+package io.spine.protodata.cli.app
 
-import com.google.auto.service.AutoService;
-import com.google.protobuf.ExtensionRegistry;
-import io.spine.option.OptionsProvider;
-import io.spine.protodata.test.MetaProto;
+import com.google.protobuf.ExtensionRegistry
+import io.spine.code.proto.FileSet
+import io.spine.option.OptionsProvider
+import io.spine.protobuf.outerClass
+import io.spine.protobuf.registerAllExtensions
 
 /**
- * Registers custom proto options introduced by this library.
+ * An [OptionsProvider] which provides all the options defined in a set of Protobuf files.
```

Comment:
As discussed vocally, please extend the description of this provider, so that next time we could understand the use-case faster.